stripe/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -183,16 +183,16 @@ data class PaymentMethodCreateParams internal constructor(
 
     internal enum class Type(internal val code: String, val hasMandate: Boolean = false) {
```

Comment:
DO we need a test for each of the payment methods, or just ideal?

Reply:
I think this is probably enough